subsys/bluetooth/host/gatt.c

```diff
@@ -2901,6 +2939,8 @@ bool bt_gatt_is_subscribed(struct bt_conn *conn,
 		attr = bt_gatt_attr_next(attr);
 	}
 
+	__ASSERT(attr, "CCC doesn't exist\n");
```

Comment:
Should be moved to right after `attr = bt_gatt_attr_next(attr);` - Since we are already accessing `attr` earlier, it has no value outside that scope.

Reply:
Yep, good catch. I added this because I actually got a segfault running the eatt babblesim test, it had forgotten to declare the CCC at the end.